### ISSUE TITLE
too-many-nested-blocks doesn't produce message when last block is the problem

### ISSUE THREAD

**Issue by david-yz-liu** [PyCQA/pylint#1088]
### Steps to reproduce
1. Run pylint on the following code:
   
   ``` python
   def func(num):
       if num > 1:
           if num > 2:
               if num > 3:
                   if num > 4:
                       if num > 5:
                           if num > 6:
                               print('greater than four')
   ```
### Current behavior

No `too-many-nested-blocks` message appears in report.
### Expected behavior

I expect to see the line R:  2, 4: Too many nested blocks (6/5) (too-many-nested-blocks).

Note: default level of max nesting is 5, and I haven't modified this. Adding any other block after this one, e.g. 

``` python
def func(num):
    if num > 1:
        if num > 2:
            if num > 3:
                if num > 4:
                    if num > 5:
                        if num > 6:
                            print('greater than four')
    if num == 0:
        print('zero')
```

will cause the message to appear.
### pylint --version output

```
No config file found, using default configuration
pylint 1.6.4,
astroid 1.4.8
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  6 2015, 01:38:48) [MSC v.1900 32 bit (Intel)]
```
